.simplecov

```diff
@@ -0,0 +1,42 @@
+# RM_INFO is set when using Rubymine.  In Rubymine, starting SimpleCov is
+# controlled by running with coverage, so don't explicitly start coverage (and
+# therefore generate a report) when in Rubymine.  This _will_ generate a report
+# whenever `rake spec` is run.
+unless ENV['RM_INFO']
```

Comment:
I use vim and Rubymine for Ruby development.  I have no idea how other IDEs coverage tools work, so yes, it's possible, but RM_INFO an environment variable that is set when Rubymine runs, so it's what I use.  We can't pick an environment variable and have all the IDEs agree on it and have it work now.  (We can petition the IDEs to agree on something, but they're not doing it now.) RM_INFO is used because it is already defined by the IDE.  If you want support for other IDEs people will have to submit PRs for whatever environment variable their IDEs use when they run rake spec or rake cucumber from inside the IDE.